docs/configuration.rst

```diff
@@ -7,107 +7,164 @@ Configuring django-csp
+    Set the ``navigate-to`` directive. A ``tuple`` or ``list``. *None*
+
+    Note: This doesn't use ``default-src`` as a fall-back.
+
 ``CSP_FORM_ACTION``
-    Set the ``FORM_ACTION`` directive. A tuple or list. *None*
-    Note: This doesn't use default-src as a fall-back.
+    Set the ``FORM_ACTION`` directive. A ``tuple`` or ``list``. *None*
+
+    Note: This doesn't use ``default-src`` as a fall-back.
+
 ``CSP_SANDBOX``
-    Set the ``sandbox`` directive. A tuple or list. *None*
-    Note: This doesn't use default-src as a fall-back.
+    Set the ``sandbox`` directive. A ``tuple`` or ``list``. *None*
+
+    Note: This doesn't use ``default-src`` as a fall-back.
+
 ``CSP_REPORT_URI``
-    Set the ``report-uri`` directive. A tuple or list. Each URI can be a
-    full or relative URI. *None*
-    Note: This doesn't use default-src as a fall-back.
+    Set the ``report-uri`` directive. A ``tuple`` or ``list`` of URIs.
+    Each URI can be a full or relative URI. *None*
+
+    Note: This doesn't use ``default-src`` as a fall-back.
+
 ``CSP_REPORT_TO``
-    Set the ``report-to`` directive. A string describing a reporting group. *None*
-    Note: This doesn't use default-src as a fall-back.
+    Set the ``report-to`` directive. A ``string`` describing a reporting
+    group. *None*
+
     See Section 1.2: https://w3c.github.io/reporting/#group
+
+    Note: This doesn't use ``default-src`` as a fall-back.
+
 ``CSP_MANIFEST_SRC``
-    Set the ``manifest-src`` directive. A tuple or list. *None*
+    Set the ``manifest-src`` directive. A ``tuple`` or ``list``. *None*
+
 ``CSP_WORKER_SRC``
-    Set the ``worker-src`` directive. A tuple or list. *None*
+    Set the ``worker-src`` directive. A ``tuple`` or ``list``. *None*
+
 ``CSP_PLUGIN_TYPES``
-    Set the ``plugin-types`` directive. A tuple or list. *None*
-    Note: This doesn't use default-src as a fall-back.
+    Set the ``plugin-types`` directive. A ``tuple`` or ``list``. *None*
+
+    Note: This doesn't use ``default-src`` as a fall-back.
+
 ``CSP_REQUIRE_SRI_FOR``
-    Set the ``require-sri-for`` directive. A tuple or list. *None*
-    Valid values: ``script``, ``style``, or both. See: require-sri-for-known-tokens_
-    Note: This doesn't use default-src as a fall-back.
+    Set the ``require-sri-for`` directive. A ``tuple`` or ``list``. *None*
+    Valid values: ``script``, ``style``, or both.
+
+    See: require-sri-for-known-tokens_
+
+    Note: This doesn't use ``default-src`` as a fall-back.
+
 ``CSP_UPGRADE_INSECURE_REQUESTS``
-    Include ``upgrade-insecure-requests`` directive. A boolean. *False*
+    Include ``upgrade-insecure-requests`` directive. A ``boolean``. *False*
+
     See: upgrade-insecure-requests_
+
 ``CSP_REQUIRE_TRUSTED_TYPES_FOR``
-    Include ``reuire-trusted-types-for`` directive. A tuple or list. *None*
+    Include ``reuire-trusted-types-for`` directive.
+    A ``tuple`` or ``list``. *None*
+
     Valid values: ``'script'``
```

Comment:
Let me change this to ` ``['script']`` `, as this setting requires a `tuple` or `list`